ir_datasets/datasets/trec_fair_2021.py

```diff
@@ -0,0 +1,179 @@
+import json
+import mwparserfromhell
+import codecs
+from typing import NamedTuple, Dict
+import ir_datasets
+from ir_datasets.util import GzipExtract, Cache, Lazy
+from ir_datasets.datasets.base import Dataset, YamlDocumentation, FilteredQueries
+from ir_datasets.formats import BaseQueries, BaseDocs, BaseQrels, TrecQrel
+from ir_datasets.indices import PickleLz4FullStore
+import pyautocorpus
+from itertools import chain
+
+
+_logger = ir_datasets.log.easy()
+
+
+NAME = 'trec_fair_2021'
+
+
+class FairTrecDoc(NamedTuple):
+    doc_id: str
+    title: str
+    text: str
+    markupfreetext: str
+    url: str
+    quality_score: str
+    geographic_locations: str
+    quality_score_disk: str
+
+
+class FairTrecQuery(NamedTuple):
+    query_id: str
+    text: str
+
+#    metadata: Dict[str, str]
+
+
+class FairTrecDocs(BaseDocs):
+    def __init__(self, name, dlc,mlc):
+        super().__init__()
+        self._name = name
+        self._dlc = dlc
+        self._mlc = mlc
+
+    def docs_iter(self):
+        return iter(self.docs_store())
+
+    def _docs_iter(self):
+        data2 = {}
+        with self._mlc.stream() as stream2:
+            
+            for line in stream2:
+                dataContx = json.loads(line) 
+                data2[dataContx["page_id"]] = dataContx
+                
+                
+                
+        
+        with self._dlc.stream() as stream1 :
+            textifier = pyautocorpus.Textifier()
```

Comment:
Can you add `pyautocorpus` to `lazy_libs` and load it like that instead? (Rather than an import at the top.) This technique means that potentially expensive imports are not always loaded, speeding up the command line interface.